modules/dreamview/backend/BUILD

```diff
@@ -24,7 +24,7 @@ cc_library(
         "//modules/dreamview/backend/point_cloud:point_cloud_updater",
         "//modules/dreamview/backend/sim_control",
         "//modules/dreamview/backend/simulation_world:simulation_world_updater",
-        "//modules/dreamview/proto:simulation_world_proto",
+        "//modules/dreamview/proto:simulation_world_cc_proto",
```

Comment:
@storypku should be working on proto renaming in master branch. Please talk with him about the timeline. If it's a long run, we can put it here :)

Because the BUILD files change very frequently everywhere, so we try to keep minimum changes in the docker_dev. Otherwise a lot of conflicts will be raised every time when we rebase.

Reply:
> @storypku should be working on proto renaming in master branch. Please talk with him about the timeline. If it's a long run, we can put it here :)
> 
> Because the BUILD files change very frequently everywhere, so we try to keep minimum changes in the docker_dev. Otherwise a lot of conflicts will be raised every time when we rebase.

Proto renaming can't be performed on the master branch as bazel version there were 0.5.3, which doesn't support py_grpc_library cc_grpc_library rules and even py_proto_library rules.

I will perform a short evaluation on proto renaming in the master branch, to check whether it can be performed there, or should we leave it here with the burden of resolving frequent rebase conflict.

Reply:
I mean just rename the cc proto target to `xx_cc_proto`. Then fix all usages (like here).
So that most parts are aligned.

Reply:
> I mean just rename the cc proto target to `xx_cc_proto`. Then fix all usages (like here).
> So that most parts are aligned.

Which, as I can see, will take a lot of pains in the master branch.

Reply:
Well, no problem, it's your call :)